modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AMQPMetaDataHandler.java

```diff
@@ -31,15 +31,26 @@
  */
 public class AMQPMetaDataHandler {
 
-    public static byte[] constructMetadata(String routingKey, ByteBuffer buf, StorableMessageMetaData originalMeataData,
+    /**
+     * This method will construct meta data while creating a copy of MessageMetadata for AMQP; after an update of the
```

Comment:
you don't have to say 'This method' 
draw example from JDK documentation it self : https://docs.oracle.com/javase/7/docs/api/java/io/File.html